### PR TITLE
dashboard showcase ruby

### PR DESCRIPTION
**Pull request by yaacov**
Translate `dashboard_showcase.sh` to ruby

### CODE REVIEW

**Comment by zeari** [comment]
@moolitayer Please review

**Comment by zeari** [comment]
@yaacov Please delete previous script

File: dashboard_showcase.rb [hunk 1]
```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env ruby
+
+def load_rails # TODO: extract to script
```
Comment (moolitayer) [inline dashboard_showcase.rb:3]: Yeah I thought about this two, but adding dependencies between scripts is a whole new level of complexity so I prefer this approach for now.

File: dashboard_showcase.rb [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  ENV['RAILS_ENV'] = ARGV[0] || 'development'
+  require './config/environment'
+end
+
+# --------------------
+# functions and consts
+# --------------------
+
+def push_metric (prov, timestamp, interval = "daily")
+  prov.metric_rollups << MetricRollup.create(
```
Comment (moolitayer) [inline dashboard_showcase.rb:13]: If you want you can save the `prov.metric_rollups <<` with adding resource_type and resource_id to the create.
I'm ok with both approaches

**Comment by moolitayer** [comment]
LFGTM @zeari

File: dashboard_showcase.rb [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+  (0 .. 24).each { |x| push_nodes_metric(prov, x.hours.ago) }
+end
+
+# --------------------
+# push metric data
+# --------------------
+
+load_rails
+
+ExtManagementSystem.all.each do |prov|
```
Comment (moolitayer) [inline dashboard_showcase.rb]: ManageIQ::Providers::ContainerManager.all.each do |prov|

**Comment by zeari** [comment]
> LFGTM @zeari

Thats the spirit

**Pull request merged**